testing/trino-tests/src/test/java/io/trino/tests/TestServer.java

```diff
@@ -275,6 +278,19 @@ public void testVersionOnCompilerFailedError()
         }
     }
 
+    @Test
+    public void testSetPathSupportByClient()
+    {
+        try (TestingTrinoClient testingClient = new TestingTrinoClient(server, testSessionBuilder().build(), Set.of())) {
```

Comment:
i think i mis-advised you.
The Session (`testSessionBuilder().build()`) defines capabilities, so when constructing `TestingTrinoClient` we don't need to pass separate set of overrides, do we?

So maybe we don't need the `io.trino.testing.AbstractTestingTrinoClient#clientCapabilitiesOverride` field, but still can modify the line:
https://github.com/trinodb/trino/blob/0c815b18566453443e769a6db3dedbe5bb1f98a0/testing/trino-testing/src/main/java/io/trino/testing/AbstractTestingTrinoClient.java#L95
because we already have the necessary information

wdyt?

Reply:
Makes sense!